timesketch/frontend/src/components/AppNavbarSecondary.vue

```diff
@@ -81,7 +87,20 @@ limitations under the License.
 <script>
 export default {
   name: 'ts-navbar-secondary',
-  props: ['currentAppContext', 'currentPage']
+  props: {
+    currentAppContext: String,
+    currentPage: String
+  },
+  methods: {
+    hasAttributeOntology: function (ontologyName) {
```

Comment:
Hmm, you could to this as a computed instead, directly on the @store.meta.attributes. This works to though. Up to you :)

Reply:
I'm not sure I can do this with a computer property, as this one is taking a parameter. The idea is to call `hasAttributeOntology('blah')` so that bits of the menubar are displayed only if the sketch has an attribute called `blah`. Can you do this with a computed property?

Reply:
Ah, I missed that you were passing a parameter. In that case, you need to use a method. 👍